docs/release-notes.md

```diff
@@ -64,7 +65,6 @@ key](https://getfedora.org/security/).
 - Document support for special mode bits and `arn` URLs _(r4e 1.1.0+)_
 - Improve rendering of spec docs on docs site
 
-
```

Comment:
random line deletion?

Reply:
I added ``` - Document `luks.s390x-device` spec _(fcos, openshift 4.14.0+)_```, then @jlebon  reviewed and suggested that "I think this section is more for doc changes for pre-existing features.". So I deleted

Reply:
I think @prestist is just referring to the fact that the net diff now shows a blank line being removed where it shouldn't.